dlc-oracle-test/src/test/scala/org/bitcoins/dlc/oracle/DLCOracleTest.scala

```diff
@@ -81,6 +81,56 @@ class DLCOracleTest extends DLCOracleFixture {
     }
   }
 
+  it must "create a ranged event and get its details" in {
+    dlcOracle: DLCOracle =>
+      val time = TimeUtil.now
+      val eventName = "ranged"
+      val start = -100
+      val stop = 100
+      val step = 5
+
+      val expectedOutcomes = start.to(stop, step).map(_.toString)
+
+      for {
+        testEventDb <-
+          dlcOracle.createNewRangedEvent(eventName, time, start, stop, step)
+        eventOpt <- dlcOracle.getEvent(testEventDb.nonce)
+      } yield {
+        assert(eventOpt.isDefined)
+        val event = eventOpt.get
+
+        assert(event.isInstanceOf[PendingEvent])
+        assert(event.eventName == eventName)
+        assert(event.outcomes == expectedOutcomes)
+        assert(event.numOutcomes == expectedOutcomes.size)
```

Comment:
nit: It might be nice to assert a hard coded value here, the answer should be 40 right?